e2e-tests/IIoTPlatform-E2E-Tests/TestHelper.cs

```diff
@@ -132,170 +133,174 @@ internal static class TestHelper {
+        /// <param name="context">Shared Context for E2E testing Industrial IoT Platform</param>
+        /// <param name="ct">Cancellation token</param>
+        public static async Task UpdateTagAsync(
+            string patch,
+            IIoTPlatformTestContext context,
+            CancellationToken ct = default
+        ) {
+            var registryManager = context.RegistryHelper.RegistryManager;
+            var twin = await registryManager.GetTwinAsync(context.DeviceConfig.DeviceId, ct);
+            await registryManager.UpdateTwinAsync(twin.DeviceId, patch, twin.ETag, ct);
+        }
+
+        /// <summary>
+        /// transfer the content of published_nodes.json file into the OPC Publisher edge module
         /// </summary>
-        /// <param name="simulatedOpcServer">Dictionary with URL of PLC-PLC as key and Content of Published Nodes files as value</param>
+        /// <param name="sourceFilePath">Source file path</param>
+        /// <param name="destinationFilePath">Destination file path</param>
         /// <param name="context">Shared Context for E2E testing Industrial IoT Platform</param>
-        public static void SavePublishedNodesFile(PublishedNodesEntryModel simulatedOpcServer, IIoTPlatformTestContext context) {
-            Assert.NotNull(simulatedOpcServer);
+        public static async Task PublishNodesInStandaloneMode(IEnumerable<PublishedNodesEntryModel> entries, IIoTPlatformTestContext context) {
+            await DeleteFileOnEdgeVM(TestConstants.PublishedNodesFullName, context);
+
+            var json = JsonConvert.SerializeObject(entries, Formatting.Indented);
+            await CreateFolderOnEdgeVM(TestConstants.PublishedNodesFolder, context);
+            using var client = CreateScpClientAndConnect(context);
+            await using var stream = new MemoryStream(Encoding.UTF8.GetBytes(json));
+            client.Upload(stream, TestConstants.PublishedNodesFullName);
 
-            PublishedNodesEntryModel[] model = new PublishedNodesEntryModel[] {simulatedOpcServer};
-            var json = JsonConvert.SerializeObject(model, Formatting.Indented);
-            context.PublishedNodesFileInternalFolder = Directory.GetCurrentDirectory() + "/published_nodes.json";
-            File.WriteAllText(context.PublishedNodesFileInternalFolder, json);
+            await SwitchToStandaloneMode(context);
         }
 
         /// <summary>
-        /// Switch to publisher standalone mode
+        /// Sets the unmanaged-Tag to "true" to enable Standalone-Mode
         /// </summary>
         /// <param name="context">Shared Context for E2E testing Industrial IoT Platform</param>
-        /// <param name="ct">Cancellation token</param>
-        public static async Task SwitchToStandaloneModeAsync(
-            IIoTPlatformTestContext context,
-            CancellationToken ct = default
-        ) {
+        /// <returns></returns>
+        public static async Task SwitchToStandaloneMode(IIoTPlatformTestContext context) {
```

Comment:
It is an `async` method, so please add `Async` in the end of method name and add `CancellationToken` parameter.